sftp/1.4/modules/ROOT/pages/sftp-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,99 @@
+= Troubleshoot SFTP Connector - Mule 4
+
+To troubleshoot Anypoint Connector for SFTP (SFTP Connector), become familiar with the information about troubleshooting connectivity, permissions and server-side issues, troubleshooting Mule application issues, and interpreting commonly thrown messages.
+
+== Troubleshoot Connectivity, Permissions and Server-Side Issues
+
+Sometimes when attempting to connect to an SFTP server you can experience problems establishing the connection because an SFTP requires a configured encrypted communication channel.
+For example, you might need to enable specific encryption cyphers or key exchange algorithms, or indicate the location of certain authorization keys.
+
+To resolve this issue:
+
+. Locate the SFTP servers configuration file to review the parameters and settings regarding the connection, encryption methods, timeouts and so forth, and correct configuration as necessary and customize the environment to suit your needs.
```

Comment:
Per our style, add a comma after "timeouts."

Reply:
"correct the configuration as necessary".

I suggest splitting this into two sentences:

.Locate the SFTP servers configuration file to review the parameters and settings regarding the connection, encryption methods, timeouts, and so forth.
.Correct the configuration as necessary, and customize the environment to suit your needs.